Resources/Prototypes/_NF/Shipyard/spirit.yml

```
@@ -12,7 +12,7 @@
   id: Spirit
   name: FSB Spirit
   description: A tiny medical search and rescue shuttle, as nimble as it is cramped. Running costs guarenteed* to be 5% lower than competing models!
```

Not really related, but while we're here...
```suggestion
  description: A tiny medical search and rescue shuttle, as nimble as it is cramped. Running costs guaranteed* to be 5% lower than competing models!
```